druid/src/widget/button.rs

```diff
@@ -52,6 +52,27 @@ impl<T: Data> Button<T> {
         }
     }
 
+    /// Create a new button with the provided label.
+    ///
+    /// Use the `.on_click` method to provide a closure to be called when the
+    /// button is clicked.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use druid::widget::Button;
+    ///
+    /// let button = Button::from_label(Label::new("Increment")).on_click(|_ctx, data: &mut u32, _env| {
+    ///     *data += 1;
+    /// });
+    /// ```
+    pub fn from_label(label: Label<T>) -> Button<T> {
+        Button {
+            label,
+            label_size: Size::ZERO,
+        }
```

Comment:
You might consider making `Button::new()` now just call this under the covers?

Reply:
Sure, why not :shrug: